### DISCUSSION TITLE
Packaged for Arch Linux

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hugetim/nbstata#34] [Show and tell]
I've packaged the project [in the AUR](https://aur.archlinux.org/packages/python-nbstata)

**Comment by hugetim** [comment]
Should we add something about this to the [docs](https://hugetim.github.io/nbstata/user_guide.html#getting-started)?

**Reply by hugetim** [reply]
Looks like maybe it's no longer there. The OP link is now broken.

**Reply by raffaelemancuso** [reply]
Yes I'm not maintaining it anymore. Feel free to fork it
